src/doc/DocDrawingThread.js

```diff
@@ -319,7 +335,7 @@ class DocDrawingThread extends DrawingThread {
      * @return {void}
      */
     position = () => {
```

Comment:
For a future PR, I would suggest changing this to `updatePosition()` or something else, since you have mixed names between variables and function names. `position` exists as a variable in this code base, and is confusing.

Reply:
will do